auto_install/install.sh

```diff
@@ -1410,7 +1562,22 @@ installOpenVPN(){
 
 	# Expect is used to feed easy-rsa with passwords
 	PIVPN_DEPS=(openvpn expect)
+
+	if [ "${PLAT}" == 'Alpine' ]
+	then
+		"${SUDO}" curl -fLo /etc/apk/keys/rsyslog@lists.adiscon.com-5a55e598.rsa.pub http://alpine.adiscon.com/rsyslog@lists.adiscon.com-5a55e598.rsa.pub
```

Comment:
Hmm, isn't there an HTTPS resource for this? Installing a public key from an untrustable source sounds like a really bad idea to me. Very confusing to me that there are still so many plain HTTP-only resources required along with this PR. In times of Let's Encrypt and many other free TLS certificate solutions that is even less understandable. However, I'm drifting off-topic...

Reply:
I'm sorry but it's the only way to install `grepcidr` on Alpine.
I'm going to get the source code directly and compile it.
To solve this, the `grepcidr` dependency should be eliminated or replaced with something similar.

Reply:
This one is about `rsyslog`. Btw doesn't the below `PIVPN_DEPS=(rsyslog)` break the actual OpenVPN installation by replacing it with `rsyslog`? And why is `rsyslog` required at all?